src/main/scala/com/scality/clueso/tools/MetadataTableCompactorTool.scala

```diff
@@ -0,0 +1,44 @@
+package com.scality.clueso.tools
+
+import java.io.File
+
+import com.scality.clueso.compact.TableFilesCompactor
+import com.scality.clueso.{CluesoConfig, SparkUtils}
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.LazyLogging
+import org.rogach.scallop._
+
+object MetadataTableCompactorTool extends LazyLogging {
+
+  class ToolConf(arguments: Seq[String]) extends ScallopConf(arguments) {
+    val applicationConfFile = trailArg[String](required = true, descr = "application configuration file")
```

Comment:
what is trailArg?

Reply:
trailing argument, so basically something that is expected after the command or after the last trailing arg